src/core/profiles/w3c/jwt.rs

```diff
@@ -98,18 +98,18 @@ impl CredentialRequestProfile for Request {
     type Response = Response;
 }
 
-#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct Response {
-    credential: String,
+    credential: CompactJWSString,
 }
 
 impl Response {
-    pub fn new(credential: String) -> Self {
+    pub fn new(credential: CompactJWSString) -> Self {
         Self { credential }
     }
     field_getters_setters![
```

Comment:
Unrelated to this PR but: if we have both getters and setters for a value that will just read/write a field, why not make the field public directly?

Reply:
I just mimicked `openidconnect` and `oauth2` APIs. I imagine they do it to have some wiggle room around breaking changes